sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationClientLiveTests.cs

```diff
@@ -202,6 +202,7 @@ public async Task GetPrebuiltModel()
         [RecordedTest]
         [TestCase(true)]
         [TestCase(false)]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/28705")]
```

Comment:
We sometimes get an empty list when listing models. Not sure if an issue on the service or client side yet.